_episodes/05-preprocessor.md

```diff
@@ -244,55 +239,64 @@ specific preprocessor which should be applied.
 >> Here is one solution to complete the challenge above using  two different
 >> preprocessors:
 >>
->>```yaml
->>
->> datasets:
->>   - {dataset: UKESM1-0-LL, project: CMIP6, exp: historical,
->>      ensemble: r1i1p1f2}  # single dataset as an example
->>
->> preprocessors:
->>   prep_map:
->>     regrid:  # apply the preprocessor to regrid
->>       target_grid: 1x1  # target resolution
->>       scheme: linear  # how to interpolate for regridding
+>>```diff
+>> --- data/recipe_example.yml
+>> +++ data/recipe_example_multi_preprocessors.yml
+>> @@ -20,23 +20,37 @@
+>>      - ukesm
 >>
->>   prep_map_land:
->>     custom_order: true  # ensure that given order of preprocessing is followed
->>     mask_landsea:    # apply a mask
->>       mask_out: sea   # mask out sea grid cells
->>     regrid:  # now apply the preprocessor to regrid
->>       target_grid: 1x1  # target resolution
->>       scheme: linear  # how to interpolate for regridding
+>>  datasets:
+>> -  - {dataset: HadGEM2-ES, project: CMIP5, exp: historical, mip: Omon, ensemble: r1i1p1, start_year: 1900, end_year: 2000}
```

Comment:
The diff shows that the recipe_example.yml starts at 1900 while on line 204 of 05-preprocessor.md there is 1970 as start year. This is not consistent please change the line 204 to 1900 or change the to 1970 in recipe_example.yml

Reply:
Maybe instead of showing recipe as code-snippet on line 199-234 we can use a link to the example recipe. Similar to exercise `Adding more preprocessor steps`. You will loose comments in the snippet that gives the student a better starting point. What do you think?

Reply:
Thanks. The start/end is corrected now. I like your suggestion and replaced the code-snippet with text.